CRM/Custom/Form/Field.php

```diff
@@ -441,21 +439,12 @@ public function buildQuickForm() {
       $attributes['help_post']
     );
 
-    // is active ?
+    $this->add('advcheckbox', 'is_required', ts('Required?'));
+    $this->addElement('advcheckbox', 'is_searchable', ts('Optimize for Search?'));
+    $this->addRadio('is_search_range', ts('Search by Range?'), [ts('No'), ts('Yes')]);
     $this->add('advcheckbox', 'is_active', ts('Active?'));
-
-    // is active ?
     $this->add('advcheckbox', 'is_view', ts('View Only?'));
 
-    // is searchable ?
-    $this->addElement('advcheckbox',
-      'is_searchable',
-      ts('Is this Field Searchable?')
-    );
-
-    // is searchable by range?
-    $this->addRadio('is_search_range', ts('Search by Range?'), [ts('No'), ts('Yes')]);
-
```

Comment:
I moved code around, to be in the same order as in the form, and also the comments were redundant

Reply:
@mlutfy looks like there's a style-lint issue

Reply:
thanks - should be fixed